### DISCUSSION TITLE
修改对照表配置的customize-los和customize-loa为false后，编译报错，是什么原因呀？

### DISCUSSION THREAD

**Discussion by noheadunhappy** [note286/xduts#42] [Q&A] [answered]
![image](https://user-images.githubusercontent.com/36783480/218471628-7ef274d0-ae1a-4b47-9905-d8cc8742bc7e.png)
![image](https://user-images.githubusercontent.com/36783480/218471652-ea7e2b9e-f4ab-4337-87ed-d58b3137707c.png)

**Comment by note286** [comment, marked as answer]
请根据[Preparation](https://github.com/note286/xduts#preparation)更新所有包。

**Comment by DuXiaoFang** [comment]
> ![image](https://user-images.githubusercontent.com/36783480/218471628-7ef274d0-ae1a-4b47-9905-d8cc8742bc7e.png) ![image](https://user-images.githubusercontent.com/36783480/218471652-ea7e2b9e-f4ab-4337-87ed-d58b3137707c.png)

请问您解决这个问题了么，是通过升级包的方法么

**Reply by note286** [reply]
使用TeX Live 2023即可。

**Reply by note286** [reply]
参考 https://github.com/note286/xduts/discussions/95 。

**Reply by DuXiaoFang** [reply]
好的好的，感谢
